games/g.yaml

```
@@ -278,6 +278,23 @@
   type: remake
   updated: 2016-06-18
   url: https://grief.ly/
+  
+- name: Grimsonland
+  type: clone
+  originals:
+  - Crimsonland
+  repo: https://github.com/BrooklynKing/Grimsonland
+  url: http://brooklynking.github.io/Grimsonland/
+  development: halted
+  status: playable
+  lang:
+  - JavaScript
+  framework:
+  - Phaser
+  license:
+  - As-is
```

```suggestion
  - MIT
```
See https://github.com/BrooklynKing/Grimsonland/blob/master/package.json#L10